com.unity.netcode.gameobjects/Tests/Runtime/RpcManyClientsTests.cs

```diff
@@ -0,0 +1,104 @@
+using System;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Unity.Netcode.RuntimeTests
+{
+    public class RpcManyClientsTest : NetworkBehaviour
+    {
+
+    }
```

Comment:
leftover? I see only 1 usage below (`AddComponent`) but maybe you meant to `AddComponent<RpcManyClientsObject>`? I'm just a bit confused (I might not spent enough time to review, sorry if that's the case)

also, do you mind inlining manual test titles into the PR description instead of their IDs/numbers for public reference?

Reply:
Good point. Removed.

Reply:
Also: updated description.